DataFormats/GeometrySurface/interface/SOARotation.h

```diff
@@ -0,0 +1,158 @@
+#pragma once
+#ifndef DataFormats_GeometrySurface_SOARotation_h
```

Comment:
is ```pragma once``` really needed given that the include guard is present?

Reply:
no. Actually I am trying to move to #pragma once and remove all those guards...

Reply:
``` #pragma once is a non-standard but widely supported preprocessor directive ```
https://en.wikipedia.org/wiki/Pragma_once

the current policy, AFAIK, is to use the include guards

Reply:
seems one wants #pragma once or include guards and not both.To start a migration towards #pragma might be best done with some discussion on the topic. [looks like its easy to find some interesting ones on google]